fab/inventory/icds

```diff
@@ -76,6 +103,8 @@ partitions='["/dev/sdb1"]'
 datavol_device='/dev/mapper/consolidated-data'
 hostname='es2'
 postgresql_max_connections=400
+pgbouncer_max_connections=3500
+pgbouncer_default_pool=390
```

Comment:
This needs to be 790 to match pg2

Reply:
pg2 is 390. Also this doesn't matter as much since it's just pgbouncer.

Reply:
Sorry. misread that